### PR TITLE
I added my blog(2)

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by samyakbvs** [comment]
file:///Users/samyakjain/Documents/GitHub/gci17.fossasia.org/index.html 
Preview 
But I don't know why its a bit different from the original site

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
Hey @samyakbvs, thanks but deployment link is one which opens the website and we can test the changes, please see https://pages.github.com.

**Comment by anshumanv** [comment]
The pages run on jekyll, so just opening the index.html file won't run the actual site, you need to run `bundle exec jekyll serve` to actually render the jekyll templating engine. For now just follow the above link and see if you learn something out of it.

**Comment by samyakbvs** [comment]
Sorry @anshumanv but I am not able to understand how to do it

**Comment by anshumanv** [comment]
Okay no issues, but please do learn it. Are you able to provide a screenshot of the changes perhaps?

File: index.html [hunk 1]
```diff
@@ -440,6 +440,17 @@ <h5 class="uppercase mb0">My Journey with Google Code-In and FOSSASIA.</h5>
                                 </div>
 
                                 <div class="row">
+                    <div class="col-md-3 col-sm-6">
+						<a href="https://samyakbvs.wordpress.com/2017/12/07/just-go-with-the-flow/" target="_blank">
+							<div class="image-tile outer-title text-center">
+								<img src="img/blog/samyak.png" height="240" width="240">
```
Comment (anshumanv) [inline index.html]: There's no image with name `satyam.png` I think your image is 6.jpg. please confirm and change accordingly.

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by anshumanv** [comment]
Please remove the other additional file as well, changes should only be to the `index.html`.

**Comment by samyakbvs** [comment]
Oh, that's a typo it is supposed to be samyak.png. Will change

**Comment by samyakbvs** [comment]
I can provide the screenshot but that screenshot won't have the exact site its a bit different

**Comment by samyakbvs** [comment]
Now is it ready to merge?

**Comment by samyakbvs** [comment]
screen shot 2017-12-08 at 7 20 56 pm
It's a bit different but I am sure that this will be in order on the original site as the image and blog have been added and not only mine everyone's image and blog have the same status

**Comment by samyakbvs** [comment]
mine but everyone's*

File: index.html [hunk 2]
```diff
@@ -440,6 +440,17 @@ <h5 class="uppercase mb0">My Journey with Google Code-In and FOSSASIA.</h5>
                                 </div>
 
                                 <div class="row">
+                    <div class="col-md-3 col-sm-6">
```
Comment (anshumanv) [inline index.html]: Correct the indentation please.

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Rest everything is good to go?

**Review by anshumanv — APPROVED** [review]

**Pull request merged**